requirements.txt

```diff
@@ -1,4 +1,5 @@
-torch>=2.0.1
+-f https://download.pytorch.org/whl/nightly/cu118
```

Comment:
Pinning this to Cuda 118 is unnecessary and detrimental to general usage with 12.2 having just released with C++20 support. I recommend finding an alternative and more dynamic method.